pages/api-directory.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import * as Sentry from "@sentry/browser";
+import "react-loader-spinner/dist/loader/css/react-spinner-loader.css";
```

Comment:
Let's get rid of react-loader-spinner if we can. Newer versions of next.js don't allow global CSS imports so we'd need to apply this to every page or do some hacks.

Reply:
(doesn't need to happen now just a note for future)

Reply:
good look